packages/generator-feathers/generators/service/index.js

```diff
@@ -20,51 +20,26 @@ module.exports = class ServiceGenerator extends Generator {
         message: 'What kind of service is it?',
```

Comment:
Are these formatting changes in this file from your linter?

Reply:
No. The CodeClimate checks originally failed because ["Function prompting has 91 lines of code (exceeds 80 allowed)."](https://codeclimate.com/github/feathersjs/feathers/pull/1057) I actually thought the formatting was better as it was. I was looking for ways to reduce the number of lines without changing behavior. I would be happy to change it back to the way it was.